src/index.ts

```diff
@@ -99,10 +94,11 @@ export async function sign(
   myRegistry.register('/types.MsgDeposit', codecs.thorchain_types.MsgDeposit)
 
   const clientOffline = await SigningStargateClient.offline(signer, { registry: myRegistry, aminoTypes: myAminoTypes, })
+  const txMessages = getMessages(tx)
 
-  if (tx.msg.length !== 1) throw new Error('support for single message signing only')
+  if (txMessages.length !== 1) throw new Error('support for single message signing only')
```

Comment:
can just inline if you want `getMessages(tx).length !== 1`

Reply:
I need the result of `getMessages(tx).length()` below. I can assign but not declare the variable inside the if condition.
I think the alternative would look something like:
```typescript
let txMessages
if((txMessages = getMessages(tx).length()) !== 1) throw new Error('support for single message signing only')
```

I'm not sure that's cleaner... unless I'm misunderstanding?

Reply:
was just thinking:
`if (getMessages(tx).length !== 1) throw new Error('support for single message signing only')`
since we don't need `txMessages` for anything and it is just kind of an awkward variable

Reply:
Ah, I see. Yeah, you're right - I'm not actually using it for anything. Inlined.